benchmarks/pytest-based/bench_algos.py

```diff
@@ -66,10 +67,11 @@
 def setup_module(module):
     """
     Trivial conversion call to force various one-time CUDA initialization
-    operations to happen outside of benchmarks.
+    operations to happen outside of benchmarks (if GPU is available).
     """
-    G = nx.karate_club_graph()
-    nxcg.from_networkx(G)
+    if cp.cuda.is_available():
+        G = nx.karate_club_graph()
+        nxcg.from_networkx(G)
```

Comment:
Should we message this to the user somehow?

```suggestion
    if cp.cuda.is_available():
        print("GPU found. Running GPU benchmarks.")
        G = nx.karate_club_graph()
        nxcg.from_networkx(G)
    else:
        print("GPU missing. Running only CPU benchmarks.")
```

Reply:
Yeah I think some kind of output like this would be useful

Think this will be useful for people running the benchmark to confirm they are covering the cases they intend

Also think it will be helpful for people reading the benchmark to spot-check that the benchmark ran as expected